packages/shared-components/src/pages/js/main.js

```diff
@@ -0,0 +1,18 @@
+let headerContext = document.createElement('onto-test-context');
```

Comment:
this is trivial, but you can rename this headerContext to testContext

Reply:
renamed

Reply:
renamed